database/phone_data/samsung-a52q.yaml

```diff
@@ -94,3 +94,18 @@ linux:
     distribution-webpage: 'https://postmarketos.org/'
     device-webpage: 'https://wiki.postmarketos.org/wiki/Samsung_Galaxy_A52_4G_(samsung-a52q)'
 
+  -
+    rom-name: 'UN1CA'
+    rom-support: true
+    rom-state: 'Official'
+    android-version: '14'
+    rom-webpage: 'https://github.com/salvogiangri/UN1CA'
+    phone-webpage: 'https://github.com/salvogiangri/UN1CA/releases/tag/2.2.1'
```

Comment:
You should use this link instead: https://github.com/salvogiangri/UN1CA/releases